Sources/SwiftSyntaxBuilder/StringConvenienceInitializers.swift

```diff
@@ -46,6 +46,12 @@ extension String: ExpressibleAsReturnClause {
   }
 }
 
+extension String: ExpressibleAsAccessPathComponent {
```

Comment:
Do we need this extensions for the test case you add in the PR?

Reply:
Yes, otherwise we need to do `ImportDecl(path: AccessPathComponent(name: "Foundation"))` etc. 

Can add this extension in another PR to separate things.